packages/server/graphql/mutations/endSprintPoker.ts

```diff
@@ -126,7 +126,8 @@ export default {
       meetingId,
       teamId,
       isKill,
-      removedTaskIds
+      removedTaskIds,
+      timelineEventId: events[0]!.id
```

Comment:
-1 Timeline events are individual to each participant, so we need to find the correct one to add here.